lms/static/scripts/frontend_apps/components/dashboard/DashboardApp.tsx

```diff
@@ -13,17 +14,23 @@ export default function DashboardApp() {
 
   return (
     <div className="flex flex-col min-h-screen gap-5">
-      <div
-        className={classnames(
-          'flex justify-center p-3 w-full',
-          'bg-white border-b shadow',
-        )}
-      >
-        <img
-          alt="Hypothesis logo"
-          src="/static/images/hypothesis-wordmark-logo.png"
-          className="h-10"
-        />
+      <div className="px-3 py-4 bg-white border-b shadow">
+        <div className="flex justify-between items-center mx-auto max-w-6xl">
+          <LogoIcon />
+          <Link
+            underline="hover"
+            variant="text"
+            href="/email/preferences"
```

Comment:
For the "Settings" link I think we should add a settings entry point soon, even if initially that is empty apart from a link to the notifications page. This will also get us to think about the URL structure. This will provide a place to add new things in future. Making the dashboard's auth work with `/email/preferences` should come first though.

Reply:
I'm trying to decide what should be included for this PR to be "ready for review".

I was checking how to make the email settings work with the dashboard auth cookie, but I think there are some considerations (for example, staff admins can access the dashboard, but not the email settings, as it requires an LTI user).

Also, maybe we should discuss it with Sean, just in case he has some opinions about this.

At the moment, this link just results in an unauthorized page, which is not great.

https://github.com/hypothesis/lms/assets/2719332/2d7455d5-8a33-4263-bbb3-5ce0f18bc478

@robertknight what would be your recommendation?

Reply:
Maybe make the link to do something like this:

```js
alert("This is not implemented yet")
```

Then we can show what the UI is supposed to look like, but internal testers won't be confused by an auth error when they click it.

Reply:
Yeah, that seems sensible.